### PR TITLE
Update Quality Declaration

### PR DESCRIPTION
**Pull request by ahcorde**
Depend on https://github.com/ros2/libyaml_vendor/pull/20

Signed-off-by: ahcorde <ahcorde@gmail.com>

### CODE REVIEW

File: libyaml_q_declaration.md [hunk 1]
```diff
@@ -85,7 +89,7 @@ Provided [doxygen documentation](https://github.com/yaml/libyaml/tree/master/doc
 
 ### Public API Documentation [3.ii]
 
-Yes, doxygen documentation is available for library .h headers.
+Yes, doxygen documentation is available for library [.h headers]([yaml.h](https://github.com/yaml/libyaml/blob/master/include/).
```
Comment (chapulina) [inline libyaml_q_declaration.md:92]: This link is malformed.

How about linking to their rendered documentation?

https://libyaml.docsforge.com/master/

File: libyaml_q_declaration.md [hunk 2]
```diff
@@ -137,10 +141,16 @@ This library does not have external dependencies.
...
+* [linux_release](https://ci.ros2.org/view/nightly/job/nightly_linux_release/lastBuild/testReport/libyaml_vendor/)
+* [mac_osx_release](https://ci.ros2.org/view/nightly/job/nightly_osx_release/lastBuild/testReport/libyaml_vendor/)
+* [windows_release](https://ci.ros2.org/view/nightly/job/nightly_win_rel/lastBuild/testReport/libyaml_vendor/)
 
 ## Security [7]
 
 ### Vulnerability Disclosure Policy [7.i]
 
-This package does not yet have a Vulnerability Disclosure Policy.
+This package does not yet have a Vulnerability Disclosure Policy. But it's defined in the vendored Quality Declaration.
```
Comment (chapulina) [inline libyaml_q_declaration.md:156]: How about

```suggestion
The `libyaml` library does not have a Vulnerability Disclosure Policy. But for ROS 2's purposes, see the policy defined in the Quality Declaration of the `libyaml_vendor` package.
```

**Review by chapulina — COMMENTED** [review]
Looks good to me after #20 is in. I just have some smaller comments below.

**Review by brawner — APPROVED** [review]
Looks good to me pending @chapulina's suggestions.

**Comment by ahcorde** [comment]
@chapulina, I can't `squash and merge` in this repo.

**Review by chapulina — APPROVED** [review]

**Review by cottsay — APPROVED** [review]

**Pull request merged**